cmd/kubeadm/app/cmd/join_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"path/filepath"
 	"testing"
 
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
It seems importing `"k8s.io/apimachinery/pkg/util/sets"` in tests yields to build errors like [that one](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/75499/pull-kubernetes-verify/1129715477703036929):
```diff
diff -Naupr -x _output /home/prow/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/BUILD /tmp/verify-bazel.STLhDZ/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/BUILD
--- /home/prow/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/BUILD	2019-05-18 11:50:23.049528531 +0000
+++ /tmp/verify-bazel.STLhDZ/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/BUILD	2019-05-18 11:51:00.467822797 +0000
@@ -99,6 +99,7 @@ go_test(
         "//staging/src/k8s.io/apimachinery/pkg/api/errors:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/runtime:go_default_library",
+        "//staging/src/k8s.io/apimachinery/pkg/util/sets:go_default_library",
         "//staging/src/k8s.io/client-go/kubernetes/fake:go_default_library",
         "//staging/src/k8s.io/client-go/testing:go_default_library",
         "//staging/src/k8s.io/client-go/tools/clientcmd:go_default_library",

Run ./hack/update-bazel.sh
```
This presumably happens because Bazel (which I'm unfamiliar with) requires this dependency to be declared. I have added it manually in `cmd/kubeadm/app/cmd/BUILD` but is there a better way to achieve this?

Reply:
Yes, there is a script - `./hack/update-bazel.sh` that will do everything for you. Actually, always use that script to update bazel deps.

Reply:
Thanks for the explanation! 👍